koku/api/report/test/ocp/tests_views.py

```diff
@@ -1521,7 +1521,7 @@ def test_execute_query_with_and_tag_group_by(self):
         self.data_generator.add_data_to_tenant()
         params = {
             "filter[resolution]": "daily",
-            "filter[time_scope_value]": "-10",
+            "filter[time_scope_value]": "-1",
             "filter[time_scope_units]": "day",
```

Comment:
I don't think these are valid. Probably needs to be `monthly`, `-1`, and `month`?